storage/google/cloud/storage/bucket.py

```diff
@@ -329,36 +356,54 @@ def bucket(self):
 
     @property
     def bucket_policy_only_enabled(self):
+        """Alias for uniform_bucket_level_access_enabled"""
+        self.uniform_bucket_level_access_enabled()
+
+    @property
+    def uniform_bucket_level_access_enabled(self):
         """If set, access checks only use bucket-level IAM policies or above.
 
         :rtype: bool
         :returns: whether the bucket is configured to allow only IAM.
         """
-        bpo = self.get("bucketPolicyOnly", {})
-        return bpo.get("enabled", False)
+        ubla = self.get("uniformBucketLevelAccess", {})
+        return ubla.get("enabled", False)
 
-    @bucket_policy_only_enabled.setter
-    def bucket_policy_only_enabled(self, value):
+    @uniform_bucket_level_access_enabled.setter
+    def uniform_bucket_level_access_enabled(self, value):
+        ubla = self.setdefault("uniformBucketLevelAccess", {})
+        ubla["enabled"] = bool(value)
+        #### THIS IS A WORKAROUND ####
         bpo = self.setdefault("bucketPolicyOnly", {})
         bpo["enabled"] = bool(value)
+        #### THIS IS A WORKAROUND ####
         self.bucket._patch_property("iamConfiguration", self)
 
+    @bucket_policy_only_enabled.setter
+    def bucket_policy_only_enabled(self, value):
+        self.uniform_bucket_level_access_enabled(value)
+
     @property
     def bucket_policy_only_locked_time(self):
-        """Deadline for changing :attr:`bucket_policy_only_enabled` from true to false.
+        """Alias for uniform_bucket_level_access."""
+        return self.uniform_bucket_level_access_locked_time()
```

Comment:
Don't call the property, just return it, e.g.:
```suggestion
        return self.uniform_bucket_level_access_locked_time
```